docs/cloud-infra-management.md

```diff
@@ -129,3 +130,19 @@ definition.json
   }
 }
 ```
+
+### generate
+
+Generates a deployment folder based on scaffolded project with a definition.json
+
+Generates a deplooyment folder of an infrastructure scaffolded project
+containing a `definition.json` that contains a `source`, `template` and
+`version` to obtain the terraform template files.
+
+It will do the following (**In Progress**):
+
+- Check if a provided project folder contains a `definition.json`
+- Check if the terraform template `source` provided has a valid remote
+  repository
+- Cache the master version of the provided `source` repository locally in
+  `~/.spk/templates`
```

Comment:
Is there ever a case where SPK config might be somewhere other than ~/.spk?  Also - how are templates differentiated?

Reply:
This may be a customization demand that we can support. This needs to be raised to the larger team though if we want to have this as a path provided through `spk init`. For simplicity, we can use the `/.spk` as this is similar to what az cli does.

Reply:
After talking to Saiya and Andre, the expectation is .spk for now, so my questions around there can be addressed in a "feature enhancement" issue that would be low pri.